server/cluster/cluster.go

```diff
@@ -491,31 +485,24 @@ func (c *RaftCluster) SetStorage(s *core.Storage) {
 }
 
 // GetOpts returns cluster's configuration.
+// There is no need a lock since it won't changed.
```

Comment:
This comment should move to the  `AddSuspectRegions`.

Reply:
This function also need.